docs/build/identity.rs/1.1/docs/getting-started/wasm.mdx

```diff
@@ -0,0 +1,341 @@
+---
+sidebar_label: Wasm
+description: Getting started with the IOTA Identity WASM Library.
+image: /img/Identity_icon.png
+keywords:
+  - WASM
+  - install
+  - npm
+  - yarn
+  - build
+  - nodejs
+  - webpack
+---
+# Getting Started with WASM
+
+## Minimum Requirements
+
+- [Node.js](https://nodejs.org/en) (>= `v16`)
+
+## Install the Library
+
+**Latest Release**: This version matches the `main` branch of this repository.
```

Comment:
```suggestion
**Latest Release**: This version matches the `main` branch of this repository. It has all the **latest features**, but as such, it **may also have undocumented breaking changes**.
```
Just in case, though I'm guessing if anything gets pushed to npm it's stable enough.

Reply:
committed a suggestion